src/m3/deserializer.ts

```diff
@@ -19,53 +19,61 @@ import {SerializedNode} from "../serialization.ts"
 import {metaConcepts, metaFeatures} from "./self-definition.ts"
 
 
+/**
+ * @return a map id -> thing with id.
+ */
+const byIdMap = <T extends { id: Id }>(ts: T[]): { [id: Id]: T } => {
+    const map: { [id: Id]: T } = {}
+    ts.forEach((t) => {
+        map[t.id] = t
+    })
+    return map
+}
+
+
 /**
  * Deserializes a metamodel that's serialized into the LIonWeb serialization JSON forma
  * as an instance of the LIonCore/M3 metametamodel, using {@link M3Concept these type definitions}.
  */
-export const deserializeMetamodel = (serializedNodes: SerializedNode[]): Metamodel => {
+export const deserializeMetamodel = (serializedNodes: SerializedNode[], ...dependentMetamodels: Metamodel[]): Metamodel => {
```

Comment:
We would need to discuss if dependencies are transitive or not

Reply:
I would say they're _not_ transitive: it's quite easy to just gather all transitive dependencies, which gives you the freedom to determine where you'd get them from, instead of having to provide a loading mechanism (load from disk, from memory, etc.).

Reply:
There's an issue for that: https://github.com/LIonWeb-org/organization/issues/50

Reply:
I think that _not_ transitive is a good idea